pandas/tests/series/test_analytics.py

```diff
@@ -1242,16 +1242,33 @@ def test_idxmin(self):
         result = s.idxmin()
         assert result == 1
 
-    def test_numpy_argmin(self):
-        # argmin is aliased to idxmin
-        data = np.random.randint(0, 11, size=10)
-        result = np.argmin(Series(data))
-        assert result == np.argmin(data)
+    @pytest.mark.skipif(PY2, reason="Buggy assertion on warning")
```

Comment:
usually having to do this means that you are NOT catching the actual assertion somewhere *else*. IOW there is another place where it IS showing the warning, but its not being caught.